README.md

```diff
@@ -486,6 +486,13 @@ Displays usage information and exits.
 L7 (HTTP/HTTPS/HTTP2) performance characterization transformation tool.
 
 ```
+
+**Example:** transform json output to fortio compatible format
```

Comment:
This is a good improvement to the documentation. Thanks!

Could we add a line saying how we got `/your/json/output/file.json`?  I'm assuming it was produced by `nighthawk_client` with `--output-format json`?

Reply:
Good idea, I'll add it!

> I'm assuming it was produced by `nighthawk_client` with `--output-format json`

Exactly as you said